chpl/chpl-resources/src/main/resources/environment.properties

```diff
@@ -170,6 +170,10 @@ testFunctionalities.restrictions=[{\
 #############################################################
 
 ######  REAL WORLD TESTING  ######
+#Date the RWT Program Started - Format is YYYY-MM-DD (format used for easier injection)
+rwtProgramStartDate=2021-09-01
+#The first Eligibility Year for RWT
+rwtProgramFirstEligibilityYear=2022
 # Date user can start entering Plan information - Format is MM/DD
```

Comment:
Should these still be here?

Reply:
Yes - they are used when determining the eligibility year.